hadoop-tools/hadoop-azure/src/site/markdown/testing_azure.md

```diff
@@ -80,7 +80,7 @@ tests against mocked storage, which is an in-memory emulation of Azure Storage.
 The integration tests are designed to test directly against an Azure storage
 service, and require an account and credentials in order to run.
 
-This is done by creating the file to `src/test/resources/azure-auth-keys.xml`
+This is done by creating the file to `src/test/resources/abfs-testrun-configs.xml`
```

Comment:
how will this work with the wasb tests? They don't get run through these shell scripts, do they?

Reply:
Basically the script runs the following command
mvn -T 1C -Dparallel-tests=abfs -Dscale -DtestsThreadCount=$threadcount verify >> "$testlogfilename"
So this is intended for ABFS